Fabric.Identity.API/Persistence/CouchDb/Stores/CouchDbResourceStore.cs

```diff
@@ -41,7 +41,7 @@ public Task<ApiResource> FindApiResourceAsync(string name)
             return DocumentDbService.GetDocument<ApiResource>(name);
```

Comment:
I didn't think we used CouchDbStore anymore, can we remove couchdb or is it still used somewhere?

Reply:
I wasn't exactly sure if we were or not. I assume we aren't, but I was mostly focused on getting IdPSS moved over without much thought to removing things we don't need.